package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "x8",
+  "version": "0.0.1",
+  "homepage": "https://new-xkit-extension.tumblr.com/",
+  "license": "GPL-3.0",
+  "browser": "inject.js",
+  "devDependencies": {
+    "webextension-polyfill": "^0.6.0",
+    "web-ext": "^4.0.0"
```

Comment:
what's web-ext?

Reply:
building for firefox (as specified in the readme :P)